doc/api/buffer.md

```diff
@@ -946,6 +994,8 @@ added: v1.1.0
 Creates and returns an [iterator] of `[index, byte]` pairs from the contents of
 `buf`.
 
+Example: Log the entire contents of a `Buffer`
```

Comment:
I think both 'contents' or 'content' are correct, but 'contents' is what "sounds" better to me.